compiler/tests/testSuite.bal

```diff
@@ -57,6 +76,26 @@ function testCompileEU(string path, string kind) returns file:Error|io:Error? {
     }
 }
 
+function importFilename(string path, string errFilename) returns string? {
+    int? index = path.lastIndexOf(".bal");
```

Comment:
use `path.endsWith(".bal")` and return if it doesn't match, this will avoid an extra level of indentation.

Reply:
function removed